Cargo.toml

```diff
@@ -58,6 +58,8 @@ default-members = [
     "ethers-solc",
 ]
 
+resolver = "2"
```

Comment:
TIL
https://doc.rust-lang.org/edition-guide/rust-2021/default-cargo-resolver.html

so if the crate does not use edition 2021 it falls back to old resolver unless the dependency uses resolver 2?

man I wonder how much time will be wasted on debugging problems this is causing

Reply:
Oh yeah there it is, virtual workspaces default to v1. Crazy